graphman-client/modules/graphman.js

```diff
@@ -7,6 +7,7 @@ const PRE_REQUEST_EXTN = utils.extension("graphman-pre-request");
 const PRE_RESPONSE_EXTN = utils.extension("graphman-pre-response");
 const http = require("http");
 const https = require("https");
+const fs = require("fs");
```

Comment:
you may use it via **utils**.
Ex: _utils.readFileBinary_

Reply:
Changed to make use of utils